etcdserver/config.go

```diff
@@ -176,7 +176,7 @@ func (c *ServerConfig) ReqTimeout() time.Duration {
 	return 5*time.Second + 2*time.Duration(c.ElectionTicks*int(c.TickMs))*time.Millisecond
 }
 
-func (c *ServerConfig) electionTimeout() time.Duration {
+func (c *ServerConfig) ElectionTimeout() time.Duration {
```

Comment:
no need to expose this, right? exposing the one in cluster.go is good enough

Reply:
@xiang90 Another question. Do we want to multiply election ticks by heartbeat ticks (as `etcdserver` does`) for `integration` package?

Reply:
i am not sure what you mean here. can you provide me the link to the code you referred to ? thanks!

Reply:
https://github.com/coreos/etcd/blob/87ad10c1552f163f541e89786597db888af8e1a6/etcdserver/config.go#L179-L181

vs

https://github.com/coreos/etcd/blob/87ad10c1552f163f541e89786597db888af8e1a6/integration/cluster.go#L611-L613

Reply:
the second one seems to be wrong.